posthog/temporal/data_imports/sources/reddit_ads/settings.py

```diff
@@ -74,7 +74,7 @@ class EndpointConfig:
         ],
         partition_keys=["created_at"],
         partition_mode="datetime",
-        partition_format="month",
+        partition_format="week",
```

Comment:
Also, we shouldn't change these without backfilling the DB btw. It'll break all existing syncs